cfme/optimize/utilization.py

```diff
@@ -25,27 +33,157 @@ def in_utilization(self):
             "Utilization",
         ]
 
+
+class UtilizationAllView(UtilizationView):
+    @property
+    def is_displayed(self):
+        return self.in_utilization and self.tree.is_displayed
+
+
+class SummaryOptionView(View):
+    show_weeks_back = BootstrapSelect(id='summ_days')
+    classification = BootstrapSelect(id='summ_tag')
+    calendar = DatePicker(id='miq_date_1')
+
+
+class DetailsOptionView(View):
+    show_weeks_back = BootstrapSelect(id='details_days')
+    classification = BootstrapSelect(id='details_tag')
+
+
+class ReportOptionView(View):
+    show_weeks_back = BootstrapSelect(id='report_days')
+    classification = BootstrapSelect(id='report_tag')
+    calendar = DatePicker(id='miq_date_2')
+
+
+class UtilizationDatastoreView(UtilizationAllView):
+    @View.nested
+    class summary(WaitTab):  # noqa
+        TAB_NAME = 'Summary'
+        options = View.nested(SummaryOptionView)
+        chart = GroupedBarChart(id='miq_chart_parent_utilts_0')
+        disk_table = Table(".//tr/th[contains(text(), 'Disk')]/ancestor::table")
+
+    @View.nested
+    class details(WaitTab):  # noqa
+        TAB_NAME = 'Details'
+        options = View.nested(DetailsOptionView)
+        disk_chart = LineChart(id="miq_chart_utiltrend_0")
+
+    @View.nested
+    class report(WaitTab):  # noqa
+        TAB_NAME = 'Report'
+        options = View.nested(ReportOptionView)
+        disk_table = Table(".//tr/th[contains(text(), 'Disk')]/ancestor::table")
+
     @property
     def is_displayed(self):
-        region = self.context["object"].appliance.region()
-        currently_selected_tree = (
-            ["Enterprise", region]
-            if self.context["object"].appliance.version > "5.11"
-            else [region]
+        is_title_matched = bool(
+            re.match(r'Datastore ".*" Utilization Trend Summary', self.title.text)
         )
-        return (
-            self.in_utilization
-            and self.tree.currently_selected == currently_selected_tree
-            and self.title.text == f'Region "{region}" Utilization Trend Summary'
+        return self.in_utilization and is_title_matched
+
+
+class UtilizationHostView(UtilizationAllView):
```

Comment:
You might hate me for this... but consider switching the naming here.

Especially when reading this module, `HostUtilizationView` and `DatastoreUtilizationView` are a lot easier to quickly tell apart then `UtilizationHostView` and `UtilizationDatastoreView`.

In the naming I would prioritize the unique entity the view is for.  This also keeps a pattern of having views like (including ones from `cfme.common.host_views` `HostDetailsView`, `HostTimelinesView`, `HostDiscoverView` - and now `HostUtilizationView`.